.azure-pipelines/gpu-benchmark.yml

```diff
@@ -26,16 +27,22 @@ jobs:
   - job: benchmarks
     timeoutInMinutes: "90"
     cancelTimeoutInMinutes: "2"
-    pool: azure-gpus-spot
+    pool: azure-jirka-spot
```

Comment:
Just leaving a reminder here that we need to revert the pool name before merge.

Reply:
this is temp solution to revert previous machines to unblock our PL CI